fdbserver/BlobManager.actor.cpp

```diff
@@ -5343,7 +5343,7 @@ ACTOR Future<Void> blobManager(BlobManagerInterface bmInterf,
                                Reference<AsyncVar<ServerDBInfo> const> dbInfo,
                                int64_t epoch) {
 	if (g_network->isSimulated()) {
-		std::string clusterId = dbInfo->get().clusterInterface.id().shortString();
+		std::string clusterId = dbInfo->get().client.clusterId.shortString();
```

Comment:
dbInfo->get().client.clusterId is unique even after ClusterController restarts.. so it's unique..